src/Core/Managed/Shared/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -202,14 +204,48 @@ protected static void BuildTelemetryProcessorChain(XElement definition, Telemetr
                     builder = builder.Use(current =>
                     {
                         var constructorArgs = new object[] { current };
-                        return (ITelemetryProcessor)LoadInstance(addElement, typeof(ITelemetryProcessor), telemetryConfiguration, constructorArgs, null);
+                        return (ITelemetryProcessor)LoadInstance(addElement, typeof(ITelemetryProcessor), null, constructorArgs, null);
                     });
                 }
             }
 
             builder.Build();
         }
 
+        protected static void LoadTelemetrySinks(XElement definition, TelemetryConfiguration telemetryConfiguration)
```

Comment:
Can you please copy this comment verbatim as a summary of the method for the future reference